app/addons/permissions/components/PermissionsSection.js

```diff
@@ -118,34 +118,50 @@ class PermissionsSection extends React.Component {
           <p className="help">
             {this.getHelp()}
             <a className="help-link" data-bypass="true" href={getDocUrl('DB_PERMISSION')} target="_blank" rel="noopener noreferrer">
-              <i className="icon-question-sign"></i>
+              <i className="fonticon-help-circled"></i>
             </a>
           </p>
         </header>
-        <div className="row-fluid">
-          <div className="span6">
+
+        <div className="row mt-2">
+          <div className="col-lg-6">
             <header>
               <h4>Users</h4>
               <p>Specify users who will have {this.props.section} access to this database.</p>
             </header>
-            <form onSubmit={this.addNames} className="permission-item-form permissions-add-user form-inline">
-              <input onChange={this.nameChange} value={this.state.newName} type="text" className="item input-small" placeholder="Username" />
-              <Button type="submit" variant="cf-primary"><i className="fonticon-plus-circled" />Add User</Button>
-            </form>
-            <ul className="list-unstyled permission-items span10">
+
+            <InputGroup>
```

Comment:
You should keep the `<form>` element with the `onSubmit` event instead of moving it to the button's `onClick`, because the former allows you to submit by pressing Enter while the focus is on the text box.